docs/sphinx/cookbook/coalesce_pool.rst

```diff
@@ -4,10 +4,49 @@
 Coalescing Pool Memory
 ======================
 
-The :class:`umpire::strategy::QuickPool` provides a
-:func:`umpire::strategy::QuickPool::coalesce` that can be used to release
+Umpire's memory pools provide a more performant way to allocate large
+amounts of memory with fewer calls to the underlying API. 
+Memory pools allow developers to allocate all needed memory 
```

Comment:
```suggestion
Memory pools allocate and manage a pool of large memory blocks.  When application
allocations are requested, the memory pool allocator will service the allocation from
these memory blocks rather than making (many) allocation requests of the underlying API
which can ...
```